doc/api/process.md

```diff
@@ -262,6 +262,20 @@ nonexistentFunc();
 console.log('This will not run.');
 ```
 
+It is possible to monitor `'uncaughtException'` events without overriding the
+default behavior to exit the process by installing a listener using the symbol
+`uncaughtExceptionMonitor`.
+
+```js
+process.on(process.uncaughtExceptionMonitor, (err, origin) => {
```

Comment:
Any particular reason for using a Symbol? I feel like just using `'uncaughtExceptionMonitor'` as a string should be fine and a bit easier?

Reply:
Reason is like in https://github.com/nodejs/node/pull/30932 to avoid any conflicts as I can't know if there is anyone out there using an event with this or that name.
Having it as an exported member helps also to avoid typos as someone would register an `undefined` type (well, type argument is not checked currently but this could be easily added).

But I'm fine with replacing it by some string.

Reply:
Unlike #30932 this PR doesn’t apply to generic event listeners and only `process`, so I would have a preference to go with a string :)

Reply:
To ensure that I understand you correct: You perfer to completly remove `process.uncaughtExceptionMonitor` and replace it by a string which is mentioned in docs only?

Reply:
done